### PR TITLE
update feature gate information

### PR DESCRIPTION
**Pull request by iheartNathan**
Description
fixes this #47996 

Issues
Closes: #47996

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/dynamic-resource-allocation.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ _build:
 stages:
   - stage: alpha
     defaultValue: false
-    fromVersion: "1.30"
+    fromVersion: "1.29"
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/dynamic-resource-allocation.md:11]: It's actually
```suggestion
    fromVersion: "1.26"
```
IIRC

**Review by sftim — COMMENTED** [review]
One error I recommend fixing

/lgtm
The docs are less wrong if we merge this.

**Review by sftim — COMMENTED** [review]
/lgtm

**Review by dipesh-rawat — COMMENTED** [review]
/approve

**Pull request merged**